tests/Formatter/CronTest.php

```diff
@@ -1,13 +1,13 @@
 <?php
 
-namespace MyBuilder\Cronos\Tests\Formatter;
+namespace Formatter;
```

Comment:
Rather than updating the namespace can we add an autoload-dev to the composer.json which should solve the issue you were seeing in phpstorm I belive

Reply:
Of course. Done